app/models/order.rb

```diff
@@ -38,4 +42,15 @@ def item_subtotal(item_id)
   def unique_items
     items.uniq
   end
+  
+  def create_items_by_store(items)
+    items_by_store = {}
+    items.each do |item, quantity|
+      items_by_store[item.store_id] = {item => quantity} if items_by_store[item.store_id].nil?
+      items_by_store[item.store_id].merge!({item => quantity}) if items_by_store[item.store_id]
+    end
+    items_by_store
+  end
+
```

Comment:
This is going to very helpful for dealing with store orders. I would love it if you could walk me through how this will work for user orders tomorrow. That part is a little unclear to me.

Reply:
will do. It's a bit different but makes sense in a multi-tenancy situation like this. We can chat about it tomorrow.